### ISSUE TITLE
Unable to parse simple Data Class with a Double

### ISSUE THREAD

**Issue by jonw2000** [cbeust/klaxon#80]
When I run the code below I get the following error:

```com.beust.klaxon.KlaxonException: Couldn't find a suitable constructor to initialize with {}```

```
data class Balance(val amount: Double)

class JsonCodecTest {

    @Test
    fun mockClientWorks() {
        val json = "{\"amount\": 1000.23}"
        val signal = Klaxon().parse<Balance>(json)
        assertEquals(signal, Balance(1000.23))
    }
}
```

Am I doing something wrong or is this a genuine issue?

Many thanks for your time.

**Comment by cbeust** [comment]
Should be fixed in 2.0.7.

**Comment by jonw2000** [comment]
Awesome, thanks
